harvester_e2e_tests/conftest.py

```diff
@@ -269,6 +275,13 @@ def pytest_collection_modifyitems(config, items):
             if 'windows_vm' in item.keywords:
                 item.add_marker(skip_windows_vm)
 
+    if config.getoption('--opensuse-image-url') == '':
+        skip_create_image_url = pytest.mark.skip(reason=(
+            'openSUSE image not specified'))
+        for item in items:
+            if 'windows_vm' in item.keywords:
```

Comment:
This line means any tests marks with `windows_vm` will be skipped if `opensuse-image-url` is empty, I think it is not reflect to the thing what we want to do.

Reply:
Good catch. I'll fix that.

Reply:
I removed that part.